.eslintrc.json

```diff
@@ -27,5 +27,14 @@
   ],
   "rules": {
     "react/react-in-jsx-scope": "off"
-  }
+  },
+  "overrides": [
+    {
+      "files": "*.js",
+      // rule overrides for js config files (next.config.js, etc.)
+      "rules": {
+        "@typescript-eslint/no-var-requires": "off"
+      }
+    }
+  ]
```

Comment:
Allows us to use `require` statements in `.js` files, which I imagine will mostly be configuration files for this project.